.github/workflows/ci.yml

```diff
@@ -41,6 +41,9 @@ jobs:
         run: cargo fmt --all -- --check
 
       - name: Run clippy
+        run: cargo clippy --workspace --all-targets -- -D warnings
+
+      - name: Run clippy all features
         run: cargo clippy --workspace --all-targets --all-features -- -D warnings
```

Comment:
I was confused by this at first since `--all-features` implies a superset but actually this is 'with tor' and the one above is 'without tor'. We could use `-p swap --features tor`. Or even just a comment to say so no one removes the clippy command above?